tests/unit_tests/dagmc/conftest.py

```diff
@@ -0,0 +1,11 @@
+
+import pytest
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_dagmc_unit_test(request):
```

Comment:
No reason this needs to be in `conftest.py` as opposed to just `test.py` since that's the only place it is used. Alternatively, you may want to consider using the pre-existing `run_in_tmpdir` fixture and then just copying dagmc.h5m to the temporary directory. That would avoid the need to cleanup files afterwards.

Reply:
I ended up using the `cdtemp` route. I wasn't sure how to get the dagmc.h5m file to the temp directory using `run_in_tmpdir` otherwise. LMK if there's a better way to do that!